Public/Src/Cache/ContentStore/App/App.config

```diff
@@ -7,6 +7,7 @@
     </connectionManagement>
   </system.net>
   <runtime>
+    <gcServer enabled="true" />

```

Comment:
Note that in Bxl's App.config we additionally set
```xml
    <Thread_UseAllCpuGroups enabled="true"/>
    <GCCpuGroup enabled="true"/>
```
Not sure what's the significance of those two parameters though.